src/main/java/me/moodcat/database/controllers/ChatDAO.java

```diff
@@ -0,0 +1,26 @@
+package me.moodcat.database.controllers;
+
+import static me.moodcat.database.entities.QChatMessage.chatMessage;
+
+import java.util.List;
+
+import javax.persistence.EntityManager;
+
+import me.moodcat.database.entities.ChatMessage;
+import me.moodcat.database.entities.Room;
+
+public class ChatDAO extends AbstractDAO<ChatMessage> {
+
+    private static final long NUMBER_OF_CHAT_MESSAGE = 10;
```

Comment:
Yes 10 is merely a dummy value. We have to see how many we want to store. As reference: Twitch does not provide a single chat message, only new ones.